src/Core/Stack.php

```diff
@@ -104,6 +174,20 @@ public function getName()
     }
 
     /**
+     * @since 1.1.0
+     *
+     * @param string $name
+     *
+     * @return $this
+     */
+    public function setName($name): self
```

Comment:
We're still with PHP 5.6, no return typehints :(

Reply:
fixed and test added